src/models/review.ts

```diff
@@ -1,2 +1,9 @@
 export default class PullRequestReview {
+    username : string;
+    state : string;
+
+    constructor(data : any) {
+        this.username = data.user.login;
+        this.state = data.state;
+    }
```

Comment:
You can define interface for `data` and then verify types for `username` and `state`.